A1_overview_and_installation.md

```diff
@@ -45,72 +45,30 @@ Installing the software will results in:
 
 The executables will be used to start the backend connecting to the robot, while the Python packages can be used for developers to create control scripts that send commands to the backend (via the shared memory) and/or read states of the robot (also via the shared memory).
 
-Note that you need a valid [MuJoCo key](https://www.roboti.us/license.html) to use the MuJoCo simulated robot.  
-
-### Procedure
-
-> 1. Download binaries or source files from MPI-IS server
-> 2. Extract files and subsequently run scripts as explained in the following for installing the necessary dependencies
-> 3. Download and move MuJoCo key to specified location
-> 4. (Optional:) Setup colcon workspace
-> 5. (Optional:) Setup SSH authentification for GitHub
-> 6. Clone project repository from GitHub
-> 7. Compile project with ```colcon```
-> 8. Source terminal with given setup script
-> 9. Run test import commands in terminal
-
 ### From tar archives
```

Comment:
If I understand correctly, there is no tar archive involved anymore, so the title should be updated?

Reply:
+1